buzsaki_lab_to_nwb/tingley_code_septal/convert_tingley_septal.py

```diff
@@ -116,3 +118,11 @@
         conversion_options=conversion_options,
```

Comment:
`iterator_opts=dict(buffer_gb=x)` is another way to make the full conversions run faster, `x=1 GB` by default, but can safely go up to about `x=max_RAM / n_jobs * 0.8` or so.